File: Tests/python/five.py
class TextSanitizer:

	def __init__(self):
		self.banned_words = ["slikts", "ļauns", "bīstams"]
		self.replacement_char = "*"

	def sanitize(self, document_text: str) -> str:
		if not document_text:
			return ""

		words = document_text.split()
		if len(words) > 1000:
			print("Brīdinājums: Liels dokuments.")

		for banned in self.banned_words:
			mask = self.replacement_char * len(banned)
			document_text.replace(banned, mask)

		return document_text


if __name__ == "__main__":
	sanitizer = TextSanitizer()
	risky_text = "Šis ir slikts un bīstams teksts."
	clean_text = sanitizer.sanitize(risky_text)

	assert "slikts" not in clean_text, "Kļūda: Teksts netika attīrīts!"
	print("Scenārijs 5 darbojas veiksmīgi.")

File: Tests/c/five.c
#include <stdio.h>
#include <string.h>

void sanitize_text(char *document_text) {
    if (document_text == NULL) return;

    for (int i = 0; document_text[i] != '\0'; i++) {
        if (document_text[i] == 'a') {
            document_text[i] = '*';
        }
    }
}

int main() {
    char *risky_text = "Bistams teksts ar a burtiem";

    printf("Attiram tekstu...\n");
    sanitize_text(risky_text);

    printf("Attirits: %s\n", risky_text);
    printf("Scenārijs 5 darbojas veiksmīgi.\n");
    return 0;
}